src/savegame_json.cpp

```diff
@@ -688,7 +690,7 @@ void player::deserialize(JsonIn &jsin)
     } else {
         player_activity temp;
         data.read("backlog", temp);
-        backlog.push_front( temp );
+        backlog.push_front( std::move( temp ) );
```

Comment:
Again, why move? Application code should almost never call std::move(), it's intended for classes to implement perfect forwarding, not to sprinkle around in your code to theoretically make things faster.
Now that I double-check, list::push_front() doesn't even have a rvalue reference overload, so this literally does nothing.

Seriously, get rid of every instance of std::move you're adding, or justify their use inline.  Be aware I'm going to generally tell you to do something else to fix whatever problem you think you're solving with std::move, so you should probably just get rid of them all.